src/test/resources/esapi/ESAPI.properties

```diff
@@ -439,6 +439,10 @@ Logger.UserInfo=true
 # Determines whether ESAPI should log the session id and client IP.
 Logger.ClientInfo=true
 
+# Determines whether ESAPI should log the prefix of [EVENT_TYPE - APPLICATION NAME].
+# If all above Logger entries are set to false and LogIgnorePrefix is true, then the output would be the same like if no ESAPI was used
+Logger.LogIgnorePrefix=true
```

Comment:
Change this property name in accordance with @jeremiahjstacey's suggestion in c**onfiguration/esapi/ESAPI.properties.**